src/styles/card.css

```
@@ -0,0 +1,157 @@
+  border: var(--card-border) solid var(--border-color);
+  border-radius: var(--card-border-radius);
+  box-sizing: border-box;
+}
+
+.card__header {
+  box-sizing: border-box;
+  width: 100%;
+  margin: var(--card-margin);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding-left: 19px;
+  padding-right: 19px;
+  padding-top: var(--card-header-top-gap);
+}
+
+.card__image {
+  height: var(--card-image-height);
+  margin: var(--card-margin);
+  padding: var(--card-padding);
+}
+
+.card__section1 {
+  display: block;
+  box-sizing: border-box;
+  width: 100%;
+}
+
+.card__title {
+  margin: var(--card-margin);
+  vertical-align: text-top;
+  padding-left: var(--min-section-gap);
+  padding-right: var(--min-section-gap);
+  padding-top: var(--card-padding-top);
+  font-size: var(--card-font-size);
+  font-weight: 500;
+  font-family: Roboto, sans-serif;
+  font-style: medium;
+  line-height: 18px;
+  color: var(--cardfont-color);
+}
+
+.card__description {
+  margin: var(--card-margin);
+  padding: var(--card-padding);
+  padding-top: 4px;
+  padding-left: var(--min-section-gap);
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: var(--card-description-grey);
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+.card__section2 {
+  padding: var( --card-padding);
+  margin: var( --card-margin);
+  padding-left: var(--min-section-gap);
+  padding-top: 16px;
+  padding-right: var(--min-section-gap);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+  width: 100%;
+}
+
+.card__stars {
+  margin: var( --card-margin);
+  padding: var( --card-padding);
+}
+
+.card__rewievs {
+  margin: var(--card-margin);
+  padding-left: 13px;
+  line-height: 14px;
+  font-size: 10px;
+  font-weight: 400;
+  color: var(--cardfont-color);
+  font-family: Roboto, sans-serif;
+}
+
+.card__footer {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-right: var(--min-section-gap);
+  padding-top: 23px;
+  padding-left: var(--min-section-gap);
+  box-sizing: border-box;
+  width: 100%;
+}
+
+.card__price {
+  margin: var( --card-margin);
+  padding: var( --card-padding);
+  font-weight: 400;
+  font-size: var( --card-font-size);
+  line-height: 18px;
+  font-family: Roboto, sans-serif;
+  color: var(--card-description-grey);
+}
+
+.card__pricetag {
+  margin: var( --card-margin);
+  padding: var(--card-padding);
+  font-family: Roboto, sans-serif;
+  font-weight: 700;
+  font-style: bold;
+  color: var(--cardfont-color);
+}
+
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+}
+
+.card__link {
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  box-sizing: border-box;
+  width: 100%;
+  padding-top: var(--min-section-gap);
+  padding-left: var(--min-section-gap);
+  padding-right: var(--min-section-gap);
+  padding-bottom: var(--min-section-gap);
```

```suggestion
  padding: var(--min-section-gap);
```
You can use shorthand properties
https://developer.mozilla.org/en-US/docs/Web/CSS/padding#syntax